### ISSUE TITLE
chip [DMG] Endpoint 0 , Cluster 0x0000_0031 not found in IncreasecluscerDataversion! (CON-1407)

### ISSUE THREAD

**Issue by ethan-jy** [espressif/esp-matter#1145]
**Describe the bug**
I copied it from the example Light to repair it as my own Light, and I can successfully connect to TH, but I found that there is the following Error。

Error1：
chip [DMG] Endpoint 0 , Cluster 0x0000_0031 not found in IncreasecluscerDataversion!
How can I fix it

Error2:
chip[sc]: The device does nossupport GetClock RealTimeMS() API: 6cEalling back to last Known cood Urc Time
I don't have the time function enabled, I'm just a light fixture, how do I turn off the time function?

**Environment**
ubuntu，matter 1.3，esp32h2

   /* Initialize the ESP NVS layer */
    nvs_flash_init();

    app_driver_init();

    app_driver_button_init();

    //创建周期任务处理周期事件
    xTaskCreate(app_cycle_task, "cycle task", 2048, nullptr, tskIDLE_PRIORITY+1, &cycle_task_handle);
    configASSERT(cycle_task_handle);

    /* Create a Matter node and add the mandatory Root Node device type on endpoint 0 */
    node::config_t node_config;
    node_t *node = node::create(&node_config, app_attribute_update_cb, app_identification_cb);

    dimmable_light::config_t light_config;
    light_config.on_off.on_off = DEFAULT_POWER;
    light_config.on_off.lighting.start_up_on_off =DEFAULT_POWER;    //nullptr;
    light_config.level_control.current_level = DEFAULT_BRIGHTNESS;
    light_config.level_control.lighting.start_up_current_level = DEFAULT_BRIGHTNESS;
    light_config.identify.identify_type =1; //light output of a lighting product
   // light_config.groups.group_name_support =0; //not supported GroupNames

    endpoint_t *endpoint = dimmable_light::create(node, &light_config, ENDPOINT_FLAG_NONE, nullptr);

    /* These node and endpoint handles can be used to create/add other endpoints and clusters. */
    if (!node || !endpoint) {
        ESP_LOGE(TAG, "Matter node creation failed");
    }

    light_endpoint_id = endpoint::get_id(endpoint);
    ESP_LOGI(TAG, "Light created with endpoint_id %d", light_endpoint_id);

    /* Mark deferred persistence for some attributes that might be changed rapidly */
    cluster_t *level_control_cluster = cluster::get(endpoint, LevelControl::Id);
    attribute_t *current_level_attribute = attribute::get(level_control_cluster, LevelControl::Attributes::CurrentLevel::Id);
    attribute::set_deferred_persistence(current_level_attribute);

    cluster::level_control::attribute::create_on_off_transition_time(level_control_cluster, 0);
    cluster::level_control::attribute::create_on_transition_time(level_control_cluster, NULL);
    cluster::level_control::attribute::create_off_transition_time(level_control_cluster, NULL);
    cluster::level_control::attribute::create_default_move_rate(level_control_cluster, 0);

#if CHIP_DEVICE_CONFIG_ENABLE_THREAD
    /* Set OpenThread platform config */
    esp_openthread_platform_config_t config = {
        .radio_config = ESP_OPENTHREAD_DEFAULT_RADIO_CONFIG(),
        .host_config = ESP_OPENTHREAD_DEFAULT_HOST_CONFIG(),
        .port_config = ESP_OPENTHREAD_DEFAULT_PORT_CONFIG(),
    };
    set_openthread_platform_config(&config);
#endif
    //::set_custom_dac_provider
    /* Matter start */
    err = esp_matter::start(app_event_cb);
    if (err != ESP_OK) {
        ESP_LOGE(TAG, "Matter start failed: %d", err);
    }

    /* Starting driver with default values */
    app_driver_light_set_defaults(light_endpoint_id);
 
 //set openthread 10
  otPlatRadioSetTransmitPower(esp_openthread_get_instance(), 10);

**Comment by wqx6** [comment]
For Error 1:
Looks like you are calling report API before `esp_matter::start()`. The ember data model is not created before `esp_matter::start()`. so please don't call report API before starting esp_matter.

For Error 2:
This error can be ignored as the device can still use a default know good epoch time to verify the matter certificate, but please don't disable the time function as it is required for matter operational certificate verification.
